test/syslogger_SUITE.erl

```diff
@@ -0,0 +1,151 @@
+-module(syslogger_SUITE).
+
+-export([all/0,
+         init_per_suite/1,
+         end_per_suite/1,
+         init_per_testcase/2,
+         end_per_testcase/2]).
+
+-export([
+    syslogger_is_used_to_log/1,
+    log_at_every_level/1,
+    log_at_custom_level/1
+]).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+-define(HANDLER_ID, sys_log).
+-define(IDENT, "mongooseim").
+-define(FACILITY, local0).
+
+all() ->
+    [syslogger_is_used_to_log,
+     log_at_every_level,
+     log_at_custom_level
+    ].
+
+init_per_suite(Config) ->
+    LoggerConfig = logger:get_primary_config(),
+    logger:set_primary_config(level, info),
```

Comment:
I'm thinking that instead of touching the primary config, which is global and god knows what `common test` is doing with it, we could set the loglevel for the specific syslog config, maybe?

Reply:
I think this line should indeed be deleted and  `mongoose_logs:set_global_loglevel(error)` could be moved to the `syslogger_is_used_to_log` testcase as only it relies on what is set here. However, the global primary config will change anyway if we use `mongoose_logs:set_global_loglevel`. Are you saying to only use `set_module_loglevel`? There is a test for custom loglevel which needs to change the global one anyway, but tbh we could get rid of it.

Reply:
I'm referring to setting loglevel to a specific handler, which is not the same than per-module or primary. Here we're testing only syslog's handler so no need to touch globals
See https://erlang.org/doc/man/logger.html#type-handler_config, the config can take a level itself. This is applied after the primary one, so if the primary loglevel allows to pass, then for this specific handler this loglevel is checked.
And with this in mind, I'm not sure we need the two tests from the other suite, `syslogger_is_used_to_log` seems to test exactly what is needed 🤔

Reply:
As you are probably right and the tests are simply duplicated, I've removed them. That leaves only a sanity check whether a handler added through `logger:add_handler` behaves as we'd expect.